src/js/actions/app-state-actions.js

```diff
@@ -116,9 +116,15 @@ export function setServerURL(url) {
 }
 
 export function showErrorNotification(message) {
-    return {
-        type: SHOW_ERROR_NOTIFICATION,
-        message
+    return (dispatch) => {
+        setTimeout(() => {
+            dispatch(hideErrorNotification(message));
```

Comment:
Doesn't really, matter, the widget only supports one notification at the same time anyway.

By looking at this, can you do this little piece of clean up : remove `message` from the action... I don't know why it has an argument for dismissing it... probably a bad copy paste.